src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift

```diff
@@ -0,0 +1,35 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import HealthCertificateToolkit
+
+extension Name {
+
+	var fullName: String {
+		var givenName = self.givenName
+		var familyName = self.familyName
+
+		if givenName?.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
+			givenName = standardizedGivenName
+		}
+
+		if familyName?.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
+			familyName = standardizedFamilyName
+		}
+
+		return [givenName, familyName]
+			.compactMap { $0 }
+			.filter { $0.trimmingCharacters(in: .whitespacesAndNewlines) != "" }
+			.joined(separator: " ")
+	}
+
+	var standardizedName: String {
+		[standardizedGivenName, standardizedFamilyName]
+			.compactMap { $0 }
+			.filter { $0.trimmingCharacters(in: .whitespacesAndNewlines) != "" }
+			.joined(separator: " ")
+	}
```

Comment:
Why not use a PersonNameComponentsFormatter() therefore?

Reply:
I considered it, but the tech spec was very specific about the name formatting so I followed it. We could think about this again for the shown name, the standardized name is a more technical name so I think we definitely should follow the spec there. Leaving it as it is for now.

Reply:
👍